client/src/components/reviewCard.jsx

```diff
@@ -1,9 +1,14 @@
 import React from 'react';
 
-const ReviewCard = ({title}) => (
-  <li>
-    <span>{title}</span>
-  </li>
+const ReviewCard = ({stars, title, text, username, date, sweepstakes}) => (
```

Comment:
No need to change now as this is a work in progress, but you'll eventually want to be passing each of these properties into stateless components.